docs/customizations/helm.rst

```diff
@@ -37,102 +37,155 @@ General Parameters
 .. list-table::
    :header-rows: 1
 
+  
    * - Name
      - Type
      - Default
      - Description
+   * - deployCR
+     - bool
+     - `false`
+     -
+   * - imagePullSecrets
+     - list
+     - `[]`
+     -
+   * - nfd.deployNodeFeatureRules
+     - bool
+     - `true`
+     -
+   * - nfd.enabled
+     - bool
+     - `true`
+     -
    * - operator.admissionController.enabled
-     - Bool
-     - False
-     - Deploy with admission controller
+     - bool
+     - `false`
+     -
    * - operator.admissionController.useCertManager
-     - Bool
-     - True
-     - Use cert-manager for generating self-signed certificate
-   * - operator.admissionController.certificate.tlsCrt
-     - String
-     - ""
-     - External TLS certificate. Ignored if cert-manager is used
-   * - operator.admissionController.certificate.tlsKey
-     - String
-     - ""
-     - External TLS private key. Ignored if cert-manager is used
-   * - nfd.enabled
-     - Bool
-     - True
-     - Deploy Node Feature Discovery
-   * - nfd.deployNodeFeatureRules
-     - Bool
-     - True
-     - Deploy Node Feature Rules to label the nodes
+     - bool
+     - `true`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[0].preference.matchExpressions[0].key
+     - string
+     - `"node-role.kubernetes.io/master"`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[0].preference.matchExpressions[0].operator
+     - string
+     - `"In"`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[0].preference.matchExpressions[0].values[0]
+     - string
+     - `""`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[0].weight
+     - int
+     - `1`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[1].preference.matchExpressions[0].key
+     - string
+     - `"node-role.kubernetes.io/control-plane"`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[1].preference.matchExpressions[0].operator
```

Comment:
These are tough to read, and not really useful

Reply:
This is not the final `helm.rst`. It will be rendered properly once the depended PR is merged. If you want a preview look at https://github.com/Mellanox/network-operator/blob/6ae08a018f019d12fc13d4630209e25cd22cc5ef/docs/charts/helm.rst.